web/src/pages/project/[projectId]/queryPlayground.tsx

```diff
@@ -0,0 +1,130 @@
+// TODO: Remove before merge
```

Comment:
This will be removed before merging.

Reply:
Done via https://github.com/langfuse/langfuse/pull/6006/commits/a0d5f1d5cd19082d32313362a179e6aa0a80b8ca.